esm_runscripts/namelists.py

```diff
@@ -126,7 +126,33 @@ def nmls_remove(mconfig):
                     value = change_entries[key]
                     if value == "remove_from_namelist":
                         namelist_removes.append((namelist, change_chapter, key))
+
+                        # the key is probably coming from esm_tools config
+                        # files or from a user runscript. It can contain lower
+                        # case, but the original Fortran namelist could be in
+                        # any case combination. Here `original_key` is coming
+                        # from the default namelist and may contain mixed case.
+                        # `key` is the processed variable my f90nml package and
```

Comment:
Mini-typo: I guess you want `by` and not `my` ;-)

Reply:
:man_facepalming: